content/en/docs/languages/java/sampling.md

```diff
@@ -0,0 +1,36 @@
+---
+title: Sampling
+weight: 80
+---
+
+[Sampling](/docs/concepts/sampling/) is a process that restricts the amount of spans that are generated by a system. The exact sampler you should use depends on your specific needs, but in general, you should make a decision at the start of a trace and allow the sampling decision to propagate to other services.
+
+A Sampler can be set on the tracer provider using the `setSampler` method, as follows:
+
+```java
+import io.opentelemetry.sdk.trace.SdkTracerProvider;
+import io.opentelemetry.sdk.trace.samplers.Sampler;
+
+public class Example {
+  public static void main(String[] args) {
+    // Configure the tracer provider with the desired sampler
+    SdkTracerProvider tracerProvider = SdkTracerProvider.builder()
+      .setSampler(Sampler.alwaysOn()) // Set to always sample traces
+      // or
+      .setSampler(Sampler.alwaysOff()) // Set to never sample traces
+      // or
+      .setSampler(Sampler.traceIdRatioBased(0.5)) // Set to sample a fraction of traces
+      .build();
+  }
+}
+```
+
+`alwaysOn` and `alwaysOff` are self-explanatory values. `alwaysOn` means that every span is sampled, while alwaysOff means that no span is sampled. When you’re getting started, or in a development environment, use `alwaysOn`.
+
+Other samplers include:
+
+- `traceIdRatioBased`, which samples a fraction of spans, based on the fraction given to the sampler. If you set `0.5`, half of all the spans are sampled.
+- `parentBased`, which uses the parent span to make sampling decisions, if present.
+  By default, the tracer provider uses a parentBased sampler with the `alwaysOn` sampler.
+
+When in a production environment, consider using the `parentBased` sampler with the `traceIdRatioBased` sampler.
```

Comment:
So maybe this is the right place to figure out the guidance, but I think it's a little more complicated than this.

Consider a scenario where a service you don't control is upstream of your service. If that service always samples, then your service will always sample too, regardless of your value set for `traceIdRatioBased`. However, if you remove the `parentBased` sampler, then you can end up with messed up traces if a downstream service doesn't also sample in the exact same way.

There's...probably a whole article on production sampling best practices we should write here.